dashboards/CockroachDB/CockroachDB-Details-Runtime.json

```diff
@@ -0,0 +1,2017 @@
+      "fontSize": "90%",
+      "gridPos": {
+        "h": 7,
+        "w": 8,
+        "x": 16,
+        "y": 21
+      },
+      "id": 15,
+      "links": [],
+      "pageSize": null,
+      "scroll": true,
+      "showHeader": true,
+      "sort": {
+        "col": 0,
+        "desc": false
+      },
+      "styles": [
+        {
+          "colorMode": null,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "dateFormat": "YYYY-MM-DD HH:mm:ss",
+          "decimals": 0,
+          "pattern": "/.*/",
+          "thresholds": [],
+          "type": "number",
+          "unit": "none"
+        }
+      ],
+      "targets": [
+        {
+          "expr": "sys_goroutines{dcos_component_name=\"CockroachDB\",host=~\"$node\"}",
+          "format": "time_series",
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "{{host}}",
+          "refId": "A",
+          "step": 240
+        }
+      ],
+      "title": "Goroutines: $node",
+      "transform": "timeseries_aggregations",
+      "type": "table"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "${DS_PROMETHEUS}",
+      "editable": true,
+      "error": false,
+      "fill": 1,
+      "grid": {},
+      "gridPos": {
+        "h": 7,
+        "w": 16,
+        "x": 0,
+        "y": 28
+      },
+      "id": 14,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 2,
+      "links": [],
+      "nullPointMode": "connected",
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [
+        {
+          "alias": "GC Pauses",
+          "yaxis": 2
+        }
+      ],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "sum(rate(sys_gc_count{dcos_component_name=\"CockroachDB\",host=~\"$node\"}[30s]))",
+          "format": "time_series",
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "GC Runs",
+          "refId": "A",
+          "step": 120
+        },
+        {
+          "expr": "sum(rate(sys_gc_pause_ns{dcos_component_name=\"CockroachDB\",host=~\"$node\"}[$rate_interval]))",
+          "format": "time_series",
+          "interval": "",
+          "intervalFactor": 2,
+          "legendFormat": "GC Pauses",
+          "metric": "",
+          "refId": "B",
+          "step": 120
+        },
+        {
+          "expr": "sys_gc_count{dcos_component_name=\"CockroachDB\",host=~\"$node\"}",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "refId": "C"
```

Comment:
Does this need a `legendFormat`? On the graph, the legend contains the expression.

Reply:
From http://docs.grafana.org/v3.1/reference/graph/#legend-values this seems to not be a configurable option. Also, let's please think of this as the bare minimum we want for ourselves during the next MWT. There are a lot of things (e.g. linking graphs) that can be done on top of these.

Reply:
Oh sorry, yes I noticed the difference. Thanks @jongiddy, will fix this immediately :)